# Identification of Regulatory Methylation Hotspots in HIV Infection through Integrated Omics Approach and Machine Learning Framework

## Overview
This study explores the interplay between transcriptomic and epigenetic alterations in chronic HIV infection, aiming to uncover regulatory methylation hotspots (RMHs) that influence immune dysfunction. Chronic HIV infection is known to disrupt immune homeostasis, and this project hypothesizes that such disruption is partly due to coordinated transcriptomic and DNA methylation changes.
Using an integrated omics approach combining microarray expression and methylation data, the study identified 85 genes under coordinated regulation. Machine learning was applied to this integrated dataset to prioritize seven key RMHs: **_LTBP4, STAT1, SULF2, SLC2A14, SNX2, DUOX1, and TUBGCP2_**. These genes were found to be functionally linked to pathways involving TGF-Î² signaling, immune suppression, and redox imbalance.

## Poster Presented in the Final Year Project Demonstration
![Poster Presentation](figures/Poster_Demonstration_Day_GIT.jpg)
